src/screens/Profile/Login/ConfirmCode.tsx

```diff
@@ -0,0 +1,61 @@
+import FullScreenHeader from '@atb/components/screen-header/full-header';
+import {StyleSheet} from '@atb/theme';
+import {useTranslation} from '@atb/translations';
+import React, {useState} from 'react';
+import {ScrollView, View} from 'react-native';
+import {LoginNavigationProp} from './';
+import * as Sections from '@atb/components/sections';
+import Button from '@atb/components/button';
+import {useAuthState} from '@atb/auth';
+
+export type ConfirmCodeProps = {
+  navigation: LoginNavigationProp;
+};
+
+export default function ConfirmCode({navigation}: ConfirmCodeProps) {
+  const {t} = useTranslation();
+  const styles = useThemeStyles();
+  const {confirmCode} = useAuthState();
+  const [code, setCode] = useState('');
+
+  const onLogin = async () => {
+    await confirmCode(code);
+    navigation.navigate('ProfileHome');
+  };
+
+  return (
+    <View style={styles.container}>
+      <FullScreenHeader title={'Logg inn'} leftButton={{type: 'back'}} />
+
+      <View style={styles.mainView}>
+        <Sections.Section>
+          <Sections.TextInput
+            label={'Kode'}
+            placeholder="Bekreftelseskode fra SMS"
+            onChangeText={setCode}
```

Comment:
Should also use phone pad here, as well as hint to the OS that this input expects an OTP

```suggestion
            onChangeText={setCode}
            keyboardType="phone-pad"
            textContentType="oneTimeCode"
```

Reply:
Fixed